assets/js/components/HostDetails/HostDetails.jsx

```diff
@@ -118,15 +118,14 @@ function HostDetails({
                 className="mx-0.5 border-green-500 border disabled:bg-slate-50 disabled:text-slate-500 disabled:border-gray-400"
```

Comment:
```suggestion
                className="mx-0.5 border-green-500 border"
```
I guess that we can get rid of the disabled styles, this button never goes disabled.

Reply:
Yeah you're right